<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_window</class>
 <widget class="QMainWindow" name="Main_window">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Graphene</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Main_window.qrc">
    <normaloff>:/graphene/icons/Graphene.xpm</normaloff>:/graphene/icons/Graphene.xpm</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionRevert"/>
    <addaction name="actionLoadFeature"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionMoveMode"/>
    <addaction name="actionSelectionMode"/>
    <addaction name="actionManipulationMode"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionResetView"/>
    <addaction name="actionRestoreView"/>
    <addaction name="actionSetView"/>
    <addaction name="actionSaveView"/>
    <addaction name="actionLoadView"/>
    <addaction name="separator"/>
    <addaction name="actionFullscreen"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFeature">
    <property name="title">
     <string>feature</string>
    </property>
    <addaction name="actionDilateLine"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
   <addaction name="menuFeature"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionClose"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="actionRevert"/>
   <addaction name="actionLoadFeature"/>
  </widget>
  <widget class="QToolBar" name="viewToolBar">
   <property name="windowTitle">
    <string>viewToolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionResetView"/>
   <addaction name="actionRestoreView"/>
   <addaction name="actionSetView"/>
   <addaction name="actionFullscreen"/>
   <addaction name="actionScreenshot"/>
   <addaction name="separator"/>
   <addaction name="actionMoveMode"/>
   <addaction name="actionSelectionMode"/>
   <addaction name="actionManipulationMode"/>
  </widget>
  <widget class="QDockWidget" name="toolboxDockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>370</width>
     <height>194</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Tools</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>350</width>
          <height>529</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QToolBox" name="toolBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>332</width>
              <height>485</height>
             </rect>
            </property>
            <attribute name="label">
             <string/>
            </attribute>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="consoleDockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>Console</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="consoleTextEdit">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Main_window.qrc">
     <normaloff>:/graphene/icons/document-open.png</normaloff>:/graphene/icons/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="Main_window.qrc">
     <normaloff>:/graphene/icons/document-close.png</normaloff>:/graphene/icons/document-close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Main_window.qrc">
     <normaloff>:/graphene/icons/document-save.png</normaloff>:/graphene/icons/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="Main_window.qrc">
     <normaloff>:/graphene/icons/document-save-as.png</normaloff>:/graphene/icons/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionRevert">
   <property name="icon">
    <iconset resource="Main_window.qrc">
     <normaloff>:/graphene/icons/document-revert.png</normaloff>:/graphene/icons/document-revert.png</iconset>
   </property>
   <property name="text">
    <string>Revert</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="icon">
    <iconset resource="Main_window.qrc">
     <normaloff>:/graphene/icons/view-fullscreen.png</normaloff>:/graphene/icons/view-fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
   <property name="toolTip">
    <string>Toggle fullscreen mode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="actionResetView">
   <property name="icon">
    <iconset resource="Main_window.qrc">
     <normaloff>:/graphene/icons/go-home.png</normaloff>:/graphene/icons/go-home.png</iconset>
   </property>
   <property name="text">
    <string>Reset View</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionRestoreView">
   <property name="icon">
    <iconset resource="Main_window.qrc">
     <normaloff>:/graphene/icons/restore-home.png</normaloff>:/graphene/icons/restore-home.png</iconset>
   </property>
   <property name="text">
    <string>Restore View</string>
   </property>
   <property name="shortcut">
    <string>Alt+Home</string>
   </property>
  </action>
  <action name="actionSetView">
   <property name="icon">
    <iconset resource="Main_window.qrc">
     <normaloff>:/graphene/icons/set-home.png</normaloff>:/graphene/icons/set-home.png</iconset>
   </property>
   <property name="text">
    <string>Set View</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Home</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Graphene</string>
   </property>
  </action>
  <action name="actionMoveMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Main_window.qrc">
     <normaloff>:/graphene/icons/transform-move.png</normaloff>:/graphene/icons/transform-move.png</iconset>
   </property>
   <property name="text">
    <string>Move Mode</string>
   </property>
  </action>
  <action name="actionSelectionMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Main_window.qrc">
     <normaloff>:/graphene/icons/edit-select.png</normaloff>:/graphene/icons/edit-select.png</iconset>
   </property>
   <property name="text">
    <string>Selection Mode</string>
   </property>
  </action>
  <action name="actionManipulationMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Main_window.qrc">
     <normaloff>:/graphene/icons/edit-node.png</normaloff>:/graphene/icons/edit-node.png</iconset>
   </property>
   <property name="text">
    <string>Manipulation Mode</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSaveView">
   <property name="text">
    <string>Save View</string>
   </property>
  </action>
  <action name="actionLoadView">
   <property name="text">
    <string>Load View</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="icon">
    <iconset resource="Main_window.qrc">
     <normaloff>:/graphene/icons/screenshot.png</normaloff>:/graphene/icons/screenshot.png</iconset>
   </property>
   <property name="text">
    <string>Screenshot</string>
   </property>
   <property name="toolTip">
    <string>Takes a screenshot of the OpenGL widget</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Print</string>
   </property>
  </action>
  <action name="actionLoadFeature">
   <property name="icon">
    <iconset resource="Main_window.qrc">
     <normaloff>:/graphene/icons/features.png</normaloff>:/graphene/icons/features.png</iconset>
   </property>
   <property name="text">
    <string>Load Feature</string>
   </property>
  </action>
  <action name="actionDilateLine">
   <property name="text">
    <string>Dilate Line</string>
   </property>
   <property name="toolTip">
    <string>Dilate Line</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Main_window.qrc"/>
 </resources>
 <connections/>
</ui>
